src/content/tooling/wrangler/commands.md

```diff
@@ -138,6 +138,17 @@ If you would like to be able to publish your code to multiple places, please see
   If you have feedback about `wrangler dev` or general questions, we will respond [here](https://github.com/cloudflare/wrangler/issues/1047).
 
 
+### tail
+
+#### Dependencies
+
+  Wrangler tail uses cloudflared under the hood. If you are already using cloudflared, be sure you have installed the latest version. Otherwise, follow the [getting started guide](https://developers.cloudflare.com/argo-tunnel/quickstart/) for Argo Tunnel.
```

Comment:
correct me if i'm wrong, but this requires a paid argo tunnel subscription, right? can we document that here?

Reply:
actually no, we use the free tunnels. eventually it would be cool to provide support for using the paid version, but i'm inclined to wait for requests.